managedtenants/bundles/bundle.py

```diff
@@ -0,0 +1,101 @@
+import subprocess
+from pathlib import Path
+
+import semver
+import yaml
+from sretoolbox.binaries import OperatorSDK
+
+from managedtenants.bundles.exceptions import BundleError
+
+
+class Bundle:
+    operator_sdk = OperatorSDK(version="1.4.2", download_path="/tmp")
```

Comment:
Class variables are gross, so if you want to initialize this once and not have it as a dependency then declare as a private module constant outside the class: `_OPERATOR_SDK = OperatorSDK(version="1.4.2", download_path="/tmp")`. Doesn't make it immutable, but makes it much more difficult to accidentally modify.

Reply:
fixed